index.html

```diff
@@ -177,4 +177,63 @@ <h4 class="g-item10">About Me</h4>
     </p>
   </section>
   <!--About Me section-->
+  <!--Form section-->
+  <section class="c-container" id="form-section">
+    <h5 class="h-form">CONTACT US</h5>
+    <P class="p-form">
+      If you have an application you are interested in developing, 
+      a feature that you need built or a project that needs coding.
+       I’d love to help with it 
+    </P>
+    <form action="#form-section" method="post">
+      <ul class="non-style">
```

Comment:
I think here you made a small mistake, you have changed the `action` with id class name. Kindly put the link as you did before.